custom/lib_portal/wizard/library_wizard.py

```diff
@@ -1,15 +1,26 @@
-from odoo import models, fields
+from odoo import api, models, fields
 
 
 class LibraryWizard(models.TransientModel):
-    _name = 'library.wizard'
-    _description = 'Wizard for Library Module'
+    _name="lib.wizard"
+    _inherit = 'stock.scrap'
```

Comment:
I don't think this inheritance is a good idea

Reply:
Yes..everything related to scrap functionality is committed wrongly .I just realised it.will commit the changes correctly now.